tls-client-pelion/mbed_app.json

```diff
@@ -0,0 +1,38 @@
+{
+    "macros": [
+        "MBED_CONF_APP_MAIN_STACK_SIZE=4096",
+        "MBEDTLS_USER_CONFIG_FILE=\"mbedtls_mbedos_pelion_config.h\"",
+        "MBEDTLS_USE_PSA_CRYPTO"
```

Comment:
To my understanding, we'll also need `MBEDTLS_PSA_CRYPTO_C` here.

Reply:
@k-stachowiak IMHO it is not yet clear how the Mbed TLS - Mbed Crypto configuration will look like in the end, so I think for now we should adhere to the way it currently works, and there we need `MBEDTLS_PSA_CRYPTO_C`.